### PR TITLE
Setup associations

### PR DESCRIPTION
**Pull request by mbdesigns1989**
- Added all the required classes.

- Added associations between classes.

### CODE REVIEW

**Review by od-c0d3r — CHANGES_REQUESTED** [review]
Asslam Alikom @mbdesigns1989 

Hi 👋🏽, While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesn't pass the linters. or the linters isn't responding.

### Try to change `github` folder to `.github` folder, notice the dot at the beginning,
Change it and Github actions should work 👍

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **Does not count** into the code reviews limit._

File: person.rb [hunk 1]
```diff
@@ -1,28 +1,37 @@
...
+  def add_rental(date, book)
+    Rental.new(date, book, self)
   end
+
+  private :of_age?
 end
+
+# person = Person.new(28, "bignameishere")
+# puts(person.parent_permission)
+# puts(person.validate_name())
```
Comment (hiromataba) [inline person.rb:37]: - [ ] Please remove the commented code, Commented code makes your code look messy so make you class cleaner by removing the commented code

**Review by hiromataba — CHANGES_REQUESTED** [review]
Hi :wave:  @mbdesigns1989 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
### To highlight :+1: 
- No linter errors :heavy_check_mark: 
- Nice code structure  :heavy_check_mark: 

## Required Changes ♻️

- [ ] Please add more description about your project to make your README very professional 

image


_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by julie-ify — APPROVED** [review]
## Hi @mbdesigns1989, 

# Highlights
- Good job implementing the required changes from the previous reviewer ✔️ 

# Status: Approved ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# Congratulations! 🎉 

Approved emoji

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**